### PR TITLE
OCD-3508 - Edit SVAP information for specific criteria in listing details

### PR DESCRIPTION
**Pull request by tmy1313**
OCD-3508

### CODE REVIEW

File: ocd-3508.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
+DROP TABLE IF EXISTS openchpl.certification_result_svap;
+DROP TABLE IF EXISTS openchpl.svap_criteria_map;
+DROP TABLE IF EXISTS openchpl.svap;
+
+CREATE TABLE openchpl.svap (
+    id bigserial NOT NULL,
+    regulatory_text_citation varchar(30) NOT NULL,
```
Comment (andlar) [inline ocd-3508.sql:7]: Might want to check on expected values for this thing. Not sure what the max length is going to be

Comment (andlar) [inline ocd-3508.sql]: Might want to check on max length for theses values. Not sure what it will be

**Review by andlar — CHANGES_REQUESTED** [review]
We need updates to the dev/data-model.sql and dev/something-triggers.sql files too

**Review by kekey1 — DISMISSED** [review]

File: ocd-3508.sql (hunk 1 above)
Comment (tmy1313) [reply ocd-3508.sql]: The largest in length in the spreadsheet is 13 chars.  I used 30 chars since that it what we used for the `criterion.number`.

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — APPROVED** [review]

**Review by andlar — CHANGES_REQUESTED** [review]
Has merge conflicts

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]
Approving this PR, but the ticket can't move to the next stage without a pending PR that removes this dummy data. We won't incorporate that pending PR yet, but we need to have it teed up to be used when it's required

File: changes/README.md [hunk 2]
```diff
@@ -1,2 +0,0 @@
-## Need to add some directions on how the updated process works
```
Comment (andlar) [inline changes/README.md]: This file shouldn't be deleted

**Review by andlar — CHANGES_REQUESTED** [review]
I think we need an update to the soft-delete file. There's a section in there where certification_result entities are soft-deleted on trigger, so the mapping here should probably be there too

Also, and maybe this is a separate conversation, but we need a PR to remove the dummy data at some point. I don't know when that will be implemented, but we'll need it eventually, and it's easier to put it in place and let it hang out now, while it's fresh in our minds, than to try to remember it later

**Review by andlar — APPROVED** [review]

**Review by kekey1 — APPROVED** [review]

**Pull request merged**